src/pages/[scheduleId]/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { ShareButton } from '@/components/common/ShareButton'
+import { MainLayout } from '@/components/layout/MainLayout'
+import { MemoList } from '@/components/screen/MemoList'
+import { USERS, SCHEDULE } from '@/constant/data'
+
+export default function Home() {
+  return (
+    <MainLayout>
+      <div className='min-h-screen flex flex-col'>
+        <main className='flex-grow'>
+          <div className='mt-20 flex flex-col items-center justify-center gap-8'>
```

Comment:
あとSP版の下部にメモリストとフッターの隙間がなくなってるので、mtじゃなくてmyでもいいと思います

```suggestion
          <div className='my-20 flex flex-col items-center justify-center gap-8'>
```